src/Components/Blazor/testassets/Wasm.Authentication.Client/Pages/Authentication.razor

```diff
@@ -0,0 +1,29 @@
+@page "/authentication/{action}"
+@using Microsoft.Extensions.Options
+
+@inject IOptions<RemoteAuthenticationOptions<ApiAuthorizationProviderOptions>> Options
+@inject SignOutSessionStateManager SignOutSessionState
+@inject NavigationManager Navigation
+
+<DefaultAuthenticationManager Action="@Action" ApplicationPaths="Options.Value.AuthenticationPaths" />
```

Comment:
Will `ApplicationPaths` appear like this in the project template too? Or does it have a way of getting the default options from DI? Seems a bit arbitrary that you have to specify this one thing.

Reply:
Removed